bothub/api/v2/evaluate/serializers.py

```diff
@@ -297,3 +309,34 @@ def filter_intent(log, intent, min_confidence, max_confidence):
             }
 
         return {"total_pages": 0, "current_page": 1, "results": []}
+
+    def get_accuracy(self, obj):
+        return obj.intent_results.accuracy
+    
```

Comment:
this line contains whitespace:

for more details: https://www.flake8rules.com/rules/W293.html